rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -100,6 +100,21 @@ void SequentialWriter::open(
 
   const auto storage_uri = format_storage_uri(base_folder_, 0);
 
+  rcpputils::fs::path db_path(base_folder_);
+  if (!db_path.is_directory()) {
```

Comment:
nitpick: You could invert the logic here to save the `else` cause.

Reply:
Sure, np.